[Merged by Bors] - Support offline mode (#391)
stackabletech/opa-operator: docs/modules/ROOT/pages/implementation-notes.adoc

Users can manage policy rules by creating, updating and deleting `ConfigMap` resources.

> The role of the https://github.com/stackabletech/opa-bundle-builder[OPA Bundle Builder] is to convert these resources to bundles (`tar.gz` files) and make them available as an HTTP endpoint.

Reviewer: Role might be confusing ;)
```suggestion
The responsibility of the https://github.com/stackabletech/opa-bundle-builder[OPA Bundle Builder] is to convert these resources to bundles (`tar.gz` files) and make them available via an HTTP endpoint.
```
